apps/system-e2e/src/support/api-tools.ts

```diff
@@ -38,3 +38,20 @@ export async function mockApi(page: Page, url: string, response: any) {
     })
   })
 }
+
```

Comment:
The code above doesn't seem to be in use, but I didn't want to remove it without asking first since it's not our code.